python/sglang/srt/layers/rotary_embedding.py

```diff
@@ -117,6 +120,16 @@ def _compute_cos_sin_cache(self) -> torch.Tensor:
         cache = torch.cat((cos, sin), dim=-1)
         return cache
 
+    def forward(self, *args, **kwargs):
+        if torch.compiler.is_compiling():
```

Comment:
ref https://github.com/sgl-project/sglang/pull/6745

Reply:
I have removed the entire `forward`, use dispatch in custom_ops.py instead.